settings.gradle

```diff
@@ -15,10 +15,11 @@
  * limitations under the License.
  */
 
-import org.gradle.util.GradleVersion
```

Comment:
This import is needed for Gradle 5.0 compatibility. Please restore it.

Reply:
Done and done!